### PR TITLE
Big-Richmedia Bid Adapter: initial release

### PR DESCRIPTION
**Pull request by mediaconsortium-develop**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```javascript
{
  'bidder': 'bigRichmedia',
  'params': {
    'placementId': '10433394'
  },
  'mediaTypes': {
    'banner': {
      'sizes': [[300, 250], [300, 600], [1800, 1000]]
    }
  },
  'adUnitCode': 'adunit-code',
  'sizes': [[300, 250], [300, 600], [1800, 1000]],
  'bidId': '30b31c1838de1e',
  'bidderRequestId': '22edbae2733bf6',
  'auctionId': '1d1a030790a475',
  'transactionId': '04f2659e-c005-4eb1-a57c-fa93145e3843'
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/3558

### CODE REVIEW

**Comment by JulieLorin** [comment]
The CI fails because of other adapters : 
image

This PR doesn't edit any files (only file addition). Anybody would know why this would fail ?

**Review by bretg — CHANGES_REQUESTED** [review]

File: modules/bigRichmediaBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import {spec as baseAdapter} from './appnexusBidAdapter.js'; // eslint-disable-line prebid/validate-imports
+
+const BIDDER_CODE = 'big-richmedia';
+
+const metadataByRequestId = {};
+
+export const spec = {
+  version: '1.4.0',
+  code: BIDDER_CODE,
+  gvlid: baseAdapter.GVLID, // use base adapter gvlid
```
[the remark this answers is not in the document]
Comment (JulieLorin) [reply modules/bigRichmediaBidAdapter.js:13]: The adapter is fully based on Xandr and will make requests using Xandr endpoint. 
Therefore, is it OK to use it ?

[the remark this answers is not in the document]
Comment (JulieLorin) [reply modules/bigRichmediaBidAdapter.js:13]: Thank you. Is there a way to follow this issue ?

[the remark this answers is not in the document]
Comment (bretg) [reply modules/bigRichmediaBidAdapter.js:13]: I'll step aside here. Xandr is willing to represent you with their GVL ID.

File: modules/bigRichmediaBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import {BANNER, VIDEO} from '../src/mediaTypes.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {spec as baseAdapter} from './appnexusBidAdapter.js'; // eslint-disable-line prebid/validate-imports
+
+const BIDDER_CODE = 'big-richmedia';
```
[the remark this answers is not in the document]
Comment (bretg) [reply modules/bigRichmediaBidAdapter.js:6]: this is still an open issue - what's your adapter code: big-richmedia or bigRichMedia?

[the remark this answers is not in the document]
Comment (JulieLorin) [reply modules/bigRichmediaBidAdapter.js:6]: The adapter code is `big-richmedia`.
The file name has been updated in https://github.com/prebid/prebid.github.io/pull/3558

[the remark this answers is not in the document]
Comment (bretg) [reply modules/bigRichmediaBidAdapter.js]: thanks. Now the actual code reviewer can do his thing. :-)

**Review by bretg — APPROVED** [review]

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**